docs/contributing.rst

```diff
@@ -39,17 +39,17 @@ Then in each shell where you're working on the client, do:
 .. code-block:: shell
 
    source ./venv/bin/activate
+   . tests/integration/_common.sh
```

Comment:
Should this use `source` instead of `.` for consistency with the above line? (Or vice versa.)